playbooks/roles/tower/tasks/bootstrap.yml

```diff
@@ -83,6 +83,7 @@
     value:
       - "/tmp"
     tower_verify_ssl: '{{ tower_verify_ssl }}'
+  when: not tower_sre_extras
```

Comment:
The variable name `tower_sre_extras ` should be removed and we can just add something like `when bootstrap == sre`